### ISSUE TITLE
MMNumberKeyboardStylePlainButtons didn't work

### ISSUE THREAD

**Issue by zeinber** [matmartinez/MMNumberKeyboard#40]
Hey, I found when I set MMNumberKeyboardStyle with MMNumberKeyboardStylePlainButtons ,it didn't work in iPhoneX but is ok in other type phone. Can you help me？

**Comment by matmartinez** [comment]
Hi @zeinber,

Unfortunately, the plain buttons style doesn’t look right on the iPhone X. This particular style was designed to extend the buttons all the way to the edges of the screen. 

But what happens if the *screen is too wide*? For instance, on an iPad or an iPhone in landscape orientation. The keys would look huge!

The same is true for the iPhone X. There’s a considerable space at bottom edge of the screen that’s needed to accommodate the home indicator. Keys would look weird. 

Believe me, I’ve tried it... and so as well UIKit designers, I’m sure. The system keyboard design was updated with this behavior after the iPhone X. The edge to edge design for number pads was left in the past. 

MMNumberKeyboard still gives you the chance to use the plain style *if applicable*. 

TL;DR: This is expected behavior meant to protect you from a funny looking keyboard. 

Have a good one!
Matías
